dataprofiler/labelers/base_model.py

```diff
@@ -329,3 +329,12 @@ def fit(self, train_data, val_data, batch_size=32, epochs=1,
         :return: None
         """
         raise NotImplementedError()
+
+    @abc.abstractmethod
+    def close(self):
+        """
+        Close the graph of the model.
+
+        :return: None
+        """
+        raise NotImplementedError()
```

Comment:
something like the regex model does not need to close and this would cause errant functionality, IMO. We should ideate more on how to implement this if it is truly necessary.

Reply:
If true, shouldn't this fail some test?

Reply:
This is new functionality so new tests would need to exist for it to fail.